src/runtime/polyfill/cloudflare.ts

```diff
@@ -0,0 +1,4 @@
+import { nextTick } from "node:process";
+import { process as _process } from "../node/process/_process";
```

Comment:
can you add a comment here about the issue with importing `../node/process/import` directly? it might be tempting for someone to refactor this in the future and run into the issues with messed up bundling we saw and can't explain.